scripts/compile.sh

```diff
@@ -22,4 +22,5 @@ get_llvm() (
 test -f "$llvm_dir/llvm-$llvm_hash"/llvm.tgz || get_llvm
 
 MLIR_DIR="$llvm_dir/lib/cmake/mlir" cmake -S . -G Ninja -B build
+# cmake -G Ninja .. -DCMAKE_BUILD_TYPE=RelWithDebInfo -DMLIR_DIR=$CUSTOM_LLVM_ROOT/lib/cmake/mlir -DLLVM_EXTERNAL_LIT=$CUSTOM_LLVM_ROOT/bin/llvm-lit -DCMAKE_C_COMPILER=clang -DCMAKE_CXX_COMPILER=clang++ 
```

Comment:
The end goal is to be able to use the script locally. Feel free to improve it directly. Also, currently the build uses gcc.
```suggestion
```

Reply:
done